control_panel_api/tests/test_views.py

```diff
@@ -519,7 +544,11 @@ def setUp(self):
     def test_list(self):
         response = self.client.get(reverse('s3bucket-list'))
         self.assertEqual(HTTP_200_OK, response.status_code)
-        self.assertEqual(len(response.data['results']), 2)
+        self.assertEqual(len(response.data['results']), 3)
+
+        response = self.client.get(
+            reverse('s3bucket-list') + '?is_data_warehouse=true')
```

Comment:
Stupid question: If the URL is `?is_data_warehouse=false`, would only return the S3 buckets objects with that flag set to `False` right?

Reply:
Yeh of course